index.html

```diff
@@ -1,41 +1,191 @@
+                    industry’s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrp</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="about-wrapper">
+            <div class="about-me">
```

Comment:
Please update your design to stick to Figma
<img width="786" alt="image" src="https://user-images.githubusercontent.com/7243132/170840845-dc0e9a4d-809d-4e4a-b6a5-ede92a8df15a.png">

<img width="786" alt="image" src="https://user-images.githubusercontent.com/7243132/170840867-fe83cacf-5cbc-4c02-b258-7d51b6b764ec.png">

Reply:
- [ ] Just as previous reviewer has said kindly Update your design as it has not yet been update